src/wovnio/wovnphp/Logger.php

```diff
@@ -112,15 +119,21 @@ public function debug($message, $context = array())
         $this->log('DEBUG', $message, $context);
     }
 
+    private function getReasonableCurrentTime()
+    {
+        $currentTime = str_replace(' ', '', microtime());
+        $currentTime = str_replace('0.', '', $currentTime);
+        return $currentTime;
+    }
+
```

Comment:
We cannot use anything related to crypto, because some environments might not support them. The timestamp based ID generation is safest.